<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntityAddDialog</class>
 <widget class="QDialog" name="EntityAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Entidade&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nomeLabel">
       <property name="text">
        <string>Nome</string>
       </property>
       <property name="buddy">
        <cstring>nomeLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nomeLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="hPLabel">
       <property name="text">
        <string>HP</string>
       </property>
       <property name="buddy">
        <cstring>hpSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mPLabel">
       <property name="text">
        <string>MP</string>
       </property>
       <property name="buddy">
        <cstring>mpSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ataqueLabel">
       <property name="text">
        <string>Ataque</string>
       </property>
       <property name="buddy">
        <cstring>ataqueSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="visibilidadeLabel">
       <property name="text">
        <string>Visibilidade</string>
       </property>
       <property name="buddy">
        <cstring>visibilidadeSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="visibilidadeSpinBox">
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="dropsLabel">
       <property name="text">
        <string>Drops</string>
       </property>
       <property name="buddy">
        <cstring>tableView</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="velocidadeLabel">
       <property name="text">
        <string>Velocidade</string>
       </property>
       <property name="buddy">
        <cstring>velocidadeSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="velocidadeSpinBox">
       <property name="minimum">
        <number>20</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="defesaLabel">
       <property name="text">
        <string>Defesa</string>
       </property>
       <property name="buddy">
        <cstring>defesaSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="defesaSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="ataqueSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="mpSpinBox">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="hpSpinBox">
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nomeLineEdit</tabstop>
  <tabstop>hpSpinBox</tabstop>
  <tabstop>mpSpinBox</tabstop>
  <tabstop>ataqueSpinBox</tabstop>
  <tabstop>defesaSpinBox</tabstop>
  <tabstop>visibilidadeSpinBox</tabstop>
  <tabstop>velocidadeSpinBox</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EntityAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EntityAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
